nbdime/webapp/templates/diffembedded.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>nbdime - diff and merge your Jupyter notebooks</title>
+
+  </head>
+
+
+  <!-- TODO: make nbdime.init() setup the forms/input user interface? -->
+
+  <body>
+
+    <div id="nbdime-header" class="nbdime-Diff">
+      <h3>Notebook Diff</h3>
+      <script id='diff-and-base' type="application/json">{{ data|tojson|safe }}</script>
+      <div id="nbdime-header-buttonrow">
+        <input id="nbdime-hide-unchanged" type="checkbox"><label for="cbox1">Hide unchanged cells</label></input>
+        <button id="nbdime-trust" style="display: none">Trust outputs</button>
+        <button id="nbdime-close" type="checkbox" style="display: none">Close tool</button>
+        <button id="nbdime-export" type="checkbox" style="display: none">Export diff</button>
+      </div>
+      <div id=nbdime-header-banner>
+        <span id="nbdime-header-base">Base</span>
+        <span id="nbdime-header-remote">Remote</span>
+      </div>
+    </div> <!-- ndime-header -->
+
+    <div id="nbdime-root" class="nbdime-root">
+    </div>
+
+    <script src="{{ nbdime_url }}"></script>
```

Comment:
You could also put the content of the script between the `<script>` tags to make the exported file truly self-contained. Or maybe have that be configurable (some flag or value for the url causes the script to be fully embedded)?

Reply:
I was very reluctant to embed nbdime.js cause it is a big file (3.2Mb) and I had a case of CI/CD in mind where I would have ~1K diffs to store. So it seemed like a waste

After your comment I realized it is just *premature* optimization

Also I realized there is a legit use case when one want to render diff as HTML and sent it over Slack or something like that

So my suggestion.
I will try to make it embedded by default. If someone is providing `url` via a command line I would use it.
How does it sound?

Reply:
I updated the pull request.
Now by default the tool will embed the script. If url will be provided then the script won't be embedded